JITpackage/Source/grb_jitpackage.c

```diff
@@ -125,7 +125,7 @@ int main (int argc, char **argv)
         // input argument is a "response file" containing the file list
 
         // open file
-        FILE *fr = fopen (argv[1]+1, "rb") ;
+        FILE *fr = fopen (argv[1]+1, "rt") ;
```

Comment:
CMake creates a file with CRLF line endings on Windows. It looks like `fgets` reads the `\r` character unless the file is opened in `"rt"` mode.
Is that a valid identifier for the mode cross-platform? (It should probably be equivalent to `"rb"` for any platform other than Windows.)

Reply:
Never mind. The actual issue was that the string returned by `fgets` includes the final `\n`.
That should be handled correctly now.